components/StoryModal.tsx

```diff
@@ -0,0 +1,33 @@
+import { Modal, Input } from '@geist-ui/react';
+import { Story } from '../redux/types';
+
+import Owners from './Owners';
+import Labels from './Labels';
+
+interface StoryModalParams {
+  story: Story;
+  isOpen: boolean;
+  close: () => void;
+}
+
+const StoryModal = ({ story, isOpen, close }: StoryModalParams): JSX.Element => {
+  if (!Boolean(story)) {
+    return <></>;
+  }
+
+  return (
+    <div>
+      <Modal open={isOpen} key={story?.id} onClose={close}>
```

Comment:
Not sure you need the conditional op here since you have a check a few lines up

Reply:
whoops i left div in